tests/test_benchmark_cli.py

```diff
@@ -79,44 +79,22 @@ def test_option_version():
 
 
 def test_option_write(file=generate_test_path('integers.ion')):
-    # make sure it reads successfully
-    file_size, result_with_gc, write_memory_usage_peak = \
-        execution_with_command(['write', file])
-
-    assert file_size == getsize(file) / BYTES_TO_MB
-    assert result_with_gc > 0
-    assert write_memory_usage_peak >= 0
+    execution_with_command(['write', file])
 
 
 def test_option_read(file=generate_test_path('integers.ion')):
```

Comment:
Quick note that for all unit tests (not only this one),

They will only make sure the execution works and don't throw any execution. However, it won't check if the benchmark results are accurate. E.g., the unit test won't test if the c extension must be faster than pure python implementation or a sample file size must be X MB.

Reply:
Is there a way to at least test that the correct options combinations are executed?

Reply:
Yep, I returned raw table data from the execution, store them in a sorted list and compare it to desired options.